packages/react-charting/docs/TestingStrategy.md

```diff
@@ -0,0 +1,119 @@
+**Details**
```

Comment:
docs are getting bundled into the release files. Exclude them by adding in .npmignore

Reply:
added